crates/jstz_cli/sandbox-params.json

```diff
@@ -7,19 +7,23 @@
     ["edpkv8EUUH68jmo3f7Um5PezmfGrRF24gnfLpH3sVNwJnV5bVCxL2n", "4000000000000"]
   ],
   "preserved_cycles": 2,
+  "consensus_rights_delay": 2,
+  "blocks_preservation_cycles": 1,
+  "delegate_parameters_activation_delay": 2,
   "blocks_per_cycle": 8,
   "blocks_per_commitment": 4,
   "nonce_revelation_threshold": 4,
   "blocks_per_stake_snapshot": 4,
   "cycles_per_voting_period": 8,
   "hard_gas_limit_per_operation": "1040000",
-  "hard_gas_limit_per_block": "2600000",
+  "hard_gas_limit_per_block": "1733333",
```

Comment:
Can you please link the source from where you got these settings?

Reply:
From https://gitlab.com/tezos/tezos, specifically I went to `lib_parameters` in `proto_alpha` and ran `dune build`